.github/actions/check-issues-are-disabled/action.yml

```
@@ -3,7 +3,7 @@ description: Checks whether the current repository has disabled the issues featu
 
 outputs:
   issues-disabled:
-    description: Whether the issues are disabled or not.
+    description: Returns a boolean value indicating whether the issues feature is disabled.
```

```suggestion
    description: Returns 'true' or 'false' indicating whether the issues feature is disabled.
```
Output values are just strings, so they need to be parsed in PS or used like you did, because there are no boolean values in GHA:
`steps.check-issues-are-disabled.outputs.issues-disabled == 'false'`